components/Coder.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Subscribe } from 'unstated'
+import ReadFileDropContainer, { DATA_URL, TEXT } from 'dropperx'
+
+import EditorContainer from '../containers/Editor'
+import Overlay from './Overlay'
+import Carbon from './Carbon'
+
+import { DEFAULT_CODE } from '../lib/constants'
+import { isImage } from '../lib/util'
+
+const to = [EditorContainer]
```

Comment:
Does this need to be a variable? Might be more readable w/o it...

Reply:
I just want to make sure that the props is equivalent as to not cause rerenders do to new array constructions